firmware/user/utils/assets.c

```diff
@@ -600,6 +600,39 @@ void ICACHE_FLASH_ATTR drawPngSequence(pngSequenceHandle* handle, int16_t xp,
     }
 }
 
+/**
+ * Draw a PNG in a sequence of PNGs
+ *
+ * @param handle The handle to draw a png from
+ * @param xp The x coordinate to draw the asset at
+ * @param yp The y coordinate to draw the asset at
+ * @param flipLR true to flip over the Y axis, false to do nothing
+ * @param flipUD true to flip over the X axis, false to do nothing
+ * @param rotateDeg The number of degrees to rotate clockwise, must be 0-359
+ * @param frame     The frame number to draw
+ * @param inv       true to invert all colors, false to draw it normally
+ */
+void ICACHE_FLASH_ATTR drawPngSequenceInv(pngSequenceHandle* handle, int16_t xp,
```

Comment:
Note to self, have `drawPngSequence()` call this to not duplicate the function.

Stretch goal: eliminate the duplicity by updating all calls to this function from all modes.